content/docs/intro/cloud-providers/azure-classic/setup.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Azure Setup
+meta_desc: This page provides an overview of how to set up and configure credentials
+           for the Pulumi Azure Provider.
+---
+
+The [Pulumi Azure provider]({{< relref "./" >}}) uses the Azure SDK to manage and provision resources.
+
+> Pulumi relies on the Azure SDK to authenticate requests from your computer to Azure. Your credentials are never sent
+> to pulumi.com.
+
+We recommend using the [Azure CLI 2.0](https://github.com/Azure/azure-cli), instead of the legacy
+[Azure xPlat CLI](https://github.com/Azure/azure-xplat-cli).  We support both, but we recommend upgrading.
+The below instructions assume that you are using the Azure CLI 2.0.
+
+## Credentials
+
+Pulumi can authenticate to Azure using a Service Principal or the Azure CLI.
+
+If you're running the Pulumi CLI locally, in a developer scenario, we recommend using the Azure CLI.  For team
+environments, particularly in CI, a Service Principal is recommended.
+
+> **Note:** Authenticating using the CLI will not work for Service Principal logins (e.g.,
+> `az login --service-principal`).  For such cases, authenticate using the Service Principal method instead.
+
+## CLI Authentication
+
+Simply login to the Azure CLI and Pulumi will automatically use your credentials:
+
+```bash
+$ az login
+The default web browser has been opened at https://login.microsoftonline.com/common/oauth2/authorize. Please continue
+the login in the web browser. If no web browser is available or if the web browser fails to open, use device code flow
+with `az login --use-device-code`.
+```
+
+Do as instructed to login.  After completed, `az login` will return and you are ready to go.
+
+> **Note:** If you're using Government, China, or German Clouds, you'll need to configure the Azure CLI to work
+> with that cloud.  Do so by running `az cloud set --name <Cloud>`, where `<Cloud>` is one of `AzureUSGovernment`,
+> `AzureChinaCloud`, or `AzureGermanCloud`.
+
+The Azure CLI, and thus Pulumi, will use the Default Subscription by default, however it is possible to override the
+subscription, by simply setting your subscription ID to the `id` output from `az account list`'s output:
+
+```bash
+$ az account list
+```
+
+Pick out the `<id>` from the list and run:
+
+```bash
+$ az account set --subscription=<id>
+```
+
+## Service Principal Authentication
```

Comment:
for some reason, this renders odd when i view ... do we need to escape a nested markdown command?

Reply:
It seems to look fine in the preview: http://pulumi-docs-origin-pr-5351-9076732a.s3-website.us-west-2.amazonaws.com/docs/intro/cloud-providers/azure-classic/setup/